src/handlers/createProduct.ts

```diff
@@ -0,0 +1,33 @@
+import { v4 as uuidv4 } from "uuid";
+
+import {HEADERS as headers, STATUS_CODES} from "../../constants";
+import {DbConnector} from "../data-layer/db-connector";
+import {APIGatewayProxyResult} from "aws-lambda";
+
+const validateProductCreationInput = (description, count, title, price) => {
+    const areAllArgumentsPresent = description && count && title && price;
+    if (!areAllArgumentsPresent) throw Error('Not all required arguments were provided (title, description, price, count)');
+};
+
+export const main = async (event): Promise<APIGatewayProxyResult> => {
+    try {
+        const { description, count, title, price } = JSON.parse(event.body);
+        const id = uuidv4();
+        validateProductCreationInput(description, count, title, price);
```

Comment:
I suppose this function should check required data and send response with 400 error, but in the result error will be 500 (statusCode: STATUS_CODES.SERVER_ERROR, in 28 line)

Reply:
Yep, that was my mistake. I thought, that if there are any validation errors, we should return 500 err.

Reply:
I suggest fixing this in a new PR. Otherwise, we will see some conflicts due to changes in the folder structure in #3.
Btw, the fix seems to be quite easy - just add returning 400 err instead of throwing an Error.